python/py-agate-sql/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 
 set base_name       agate-sql
 name                py-agate-sql
-version             0.5.2
+version             0.5.3
 python.versions     27 35 36
```

Comment:
You might want to check on whether 3.7 is now supported.

Reply:
The developer mentions support for 3.3+ and 3.6, but I have not found any mention of 3.7 specifically (same goes for csvkit and other dependencies: agate, agate-excel, agate-dbf). Some other package managers (Anaconda, Fedora) already have it available for 3.7 though. Should a subport be added if it works?